### DISCUSSION TITLE
--exclude in the example docker-compose doesnt match what the README says

### DISCUSSION THREAD

**Discussion by grendelson** [djmaze/resticker#178] [Q&A] [answered]
The example has code like:
    RESTIC_BACKUP_ARGS: >-
        --tag docker-volumes
        --exclude='some-folder/cache'
        --exclude='*.tmp'
        --verbose

However the description of CONFIGURATION OPTIONS in the README says:

RESTIC_BACKUP_ARGS - If specified restic backup is run with the given arguments, e.g. for tags, exclude definitions, or verbose logging: --tag docker-volumes --exclude-file 'exclude.txt' --verbose. **Make sure not to use the = form of assignment, but use spaces between parameter and value.** See the restic backup documentation for available options

So when I copied/pasted then edited the example - I had the --exclude='path/to/folder' and  the excludes seem to have been ignored since i ran out of space when I should have only backed up a few GB or less.

I was unable to find ANY way to get the --exclude lines to take a space in the path 'path/to space' threw errors.  "", \ <using a backslash - all failed - I see from a few years ago the solution was to NOT use the " ' " single quote character?  
Is this still the case - if so there is not note of that in the Readme

**Comment by grendelson** [comment, marked as answer]
I might have answered my own questions - but the README & the examples are wildly out of date:


      RESTIC_BACKUP_ARGS: >-
        --tag docker-volumes
        --exclude varlib/
        --exclude calibre/Calibre\ Library
        --exclude /mnt/**/cache
        --exclude *.tmp
        --verbose

A snippet above - this Worked! But following the examples it looked like 



      RESTIC_BACKUP_ARGS: >-
        --tag docker-volumes
        --exclude='varlib/'
        --exclude='calibre/Calibre Library'
        --exclude='/mnt/**/cache'
        --exclude='*.tmp'
        --verbose

The spaces didn't work as noted back in 2021 - and with the '=' sign all the excludes were ignored.

**Reply by djmaze** [reply]
You are right. Thanks for the catch and sorry for the late reaction. The fix is in #187

**Comment by grendelson** [comment]
Sorry - I reopened so others could find this - and so it can be  fixed in the project if need be ? ( the README and the example docker-compose.yml)
